File: JavaScript/Math-Array-String/30.js
// 1662. Check If Two String Arrays are Equivalent

/**
 * @param {string[]} word1
 * @param {string[]} word2
 * @return {boolean}
 */
var arrayStringsAreEqual = function(word1, word2) {
    // Concatenate the strings in each array using join
    const str1 = word1.join('');
    const str2 = word2.join('');
    
    // Compare the concatenated strings
    return str1 === str2;
};

File: Python/Math-Array-String/30.py
# 1662. Check If Two String Arrays are Equivalent

from typing import List

class Solution:
    def arrayStringsAreEqual(self, word1: List[str], word2: List[str]) -> bool:
        # Concatenate all the strings in each array
        str1 = ''.join(word1)
        str2 = ''.join(word2)
        
        # Compare the concatenated strings
        return str1 == str2
    


sol = Solution()
output = sol.arrayStringsAreEqual(["ab", "c"], ["a", "bc"])
print(output)